### DISCUSSION TITLE
error starting to plot

### DISCUSSION THREAD

**Discussion by ShadowFku** [ericaltendorf/plotman#447] [Q&A]
Hello,

Error ate starting to plot loop

help please! :)

```
...starting plot loop
Traceback (most recent call last):
  File "./plotman", line 8, in <module>
    sys.exit(main())
  File "/home/server/.local/lib/python3.8/site-packages/plotman/plotman.py", line 145, in main
    wait_reason = manager.maybe_start_new_plot(cfg.directories, cfg.scheduling, cfg.plotting)
  File "/home/server/.local/lib/python3.8/site-packages/plotman/manager.py", line 163, in maybe_start_new_plot
    p = subprocess.Popen(plot_args,
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'chia'
```

**Comment by altendky** [comment]
This means plotman can't find the chia executable.  How to correct this depends on how you installed chia.

**Reply by stephendt** [reply]
I have this exact same issue unfortunately - struggling to understand what I have done wrong. Installed chia via the CLI as per the Chia wiki, and initialised it via . ./activate. Installed the experimental GUI via the deb file and chia is activated, still not working. I'm baffled.

**Reply by calarndt** [reply]
you must add the dir where the chia binary lives to your path: on my system that is:
/home/chia/chia-blockchain/venv/bin/chia
by doing this:
PATH=$PATH:/home/chia/chia-blockchain/venv/bin/
your chia environment must be activated ie:
cd chia-blockchain
. ./activate

**Reply by altendky** [reply]
Don't install both via the deb and via a git clone.  Then you have two copies of chia to get confused about.  Also, no need to modify your path.  That is roughly all activating the venv does.
